src/components/Input/index.test.tsx

```diff
@@ -0,0 +1,189 @@
+import {
+  render,
+  screen,
+  fireEvent,
+  within,
+  cleanup,
+} from "@testing-library/react";
+import Input from "./index";
+import { Suggestion, SuggestionList } from "./styledComponents";
+
+describe("Input component", () => {
+  it("should allow letters to be inputted", () => {
+    render(
+      <Input
+        suggestions={[
+          "Alligator",
+          "Bask",
+          "Crocodilian",
+          "Death Roll",
+          "Eggs",
+          "Jaws",
+          "Reptile",
+          "Solitary",
+          "Tail",
+          "Wetlands",
+        ]}
+      />
+    );
+
+    let input = screen.getByLabelText("search-input") as HTMLInputElement;
+    expect(input.value).toBe("");
+    fireEvent.change(input, { target: { value: "Good Day" } });
+    expect(input.value).toBe("Good Day");
+  });
+
+  it("should show suggestions list matching input target", () => {
+    render(
+      <Input
+        suggestions={[
+          "Alligator",
+          "Bask",
+          "Crocodilian",
+          "Death Roll",
+          "Eggs",
+          "Jaws",
+          "Reptile",
+          "Solitary",
+          "Tail",
+          "Wetlands",
+        ]}
+      />
+    );
+    let input = screen.getByLabelText("search-input") as HTMLInputElement;
+
+    expect(input.value).toBe("");
+    fireEvent.change(input, { target: { value: "e" } });
+    expect(input.value).toBe("e");
+
+    const list = screen.getByRole("list");
+    const { getAllByRole } = within(list);
+    const items = getAllByRole("listitem");
+    expect(items.length).toBe(4);
```

Comment:
We can also check the `items` length before typing `e` to test if there is no items.

Reply:
I can check the items length and list because they are not in the document

Reply:
Right, you could then expect there is no list in the document ;)